README.md

```diff
@@ -1 +1,35 @@
-# My-Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> HTML/CSS & JavaScript capstone project - Conference page.
+
+
```

Comment:
- [ ] This is one of your very first showcase projects for your portfolio, is necessary to make it looks the best, please make sure to add a proper description to this project in this section to guide your future visitants